src/System.Net.Requests/tests/System.Net.Requests.Tests.csproj

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Windows_Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{851A40FE-7F07-415D-8592-5FE2096E84D3}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <StringResourcesPath>..\src\Resources\Strings.resx</StringResourcesPath>
+  </PropertyGroup>
```

Comment:
A bunch of these tests that actually try to call out over the network are going to fail right now on Unix, correct?  If so, seems like we have two options.  We could either annotate each test we expect to fail right now as [ActiveIssue(somenumber, PlatformID.AnyUnix)] such that we only run on it on Windows, or we could add an entry to the tests .csproj that'll have the CI server skip this test project entirely (including tests that would actually pass on Unix today), e.g. https://github.com/dotnet/corefx/blob/master/src/Microsoft.Win32.Registry/tests/Microsoft.Win32.Registry.Tests.csproj#L11 .